src/main/asciidoc/configuration.adoc

```diff
@@ -23,7 +23,7 @@ As a result, the library can be included like any other maven dependency:
 
 This will pull in several dependencies, including the underlying Couchbase Java SDK, common Spring dependencies and also Jackson as the JSON mapping infrastructure.
 
-You can also grab snapshots from the https://repo.spring.io/libs-snapshot[spring snapshot repository] and milestone releases from the https://repo.spring.io/libs-milestone[milestone repository].
+You can also grab snapshots from the https://repo.spring.io/ui/repos/tree/General/libs-snapshot/org/springframework/data/spring-data-couchbase[spring snapshot repository] ( \https://repo.spring.io/libs-snapshot ) and milestone releases from the https://repo.spring.io/ui/repos/tree/General/libs-milestone/org/springframework/data/spring-data-couchbase[spring milestone repository] ( \https://repo.spring.io/libs-milestone ).
```

Comment:
do these links really work when put into a maven repository config? Since wen I click on them I get to the jfrug UI

Reply:
The old urls also redirect to the jfrog UI, but give 404.  The urls in parentheses are the ones to plug into the maven repository config.